From e44b6a8f5fa28cb2872e3c19bb8a205bb5bfc281 Mon Sep 17 00:00:00 2001
From: lhchavez <lhchavez@lhchavez.com>
Date: Fri, 7 Jan 2022 14:04:31 -0800
Subject: [PATCH] Change the default polling host to `gp-v2.replit.com` (#113)

We switched the host a few days ago, so it's better to reflect reality.

This changes the default polling host to `gp-v2.replit.com`.
---
 CHANGELOG.md        | 4 ++++
 src/util/helpers.ts | 2 +-
 2 files changed, 5 insertions(+), 1 deletion(-)

diff --git a/CHANGELOG.md b/CHANGELOG.md
index 7a9e056..2060e69 100644
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,7 @@
+#### v7.3.1
+
+Changed the default polling host to gp-v2.replit.com.
+
 #### v7.3.0
 
 Added an optional `pollingHost` field to `ConnectOptions`. This allows callers to override the polling host from gp-v2.herokuapp.com to something else when the polling fallback is used.
diff --git a/src/util/helpers.ts b/src/util/helpers.ts
index d88af79..0c67474 100644
--- a/src/util/helpers.ts
+++ b/src/util/helpers.ts
@@ -61,7 +61,7 @@ export function getConnectionStr(
 ): string {
   const gurl = urllib.parse(connectionMetadata.gurl);
   if (isPolling) {
-    const host = pollingHost ?? 'gp-v2.herokuapp.com';
+    const host = pollingHost ?? 'gp-v2.replit.com';
     gurl.hostname = host;
     gurl.host = host;
     gurl.pathname = `/wsv2/${connectionMetadata.token}/${encodeURIComponent(
